tests/test_data_prep_scripts.py

```diff
@@ -37,13 +37,13 @@ def test_download_script_from_api():
 
 def test_json_script_from_api():
     # test calling it directly
-    path = os.path.join(os.getcwd(), 'my-copy-c4-3')
+    path = os.path.join(os.getcwd(), 'my-copy-arxiv-1')
```

Comment:
why do we have these changes?

Reply:
The test is creating a sample arxiv dataset but the folder is named c4. just found the naming mistake when I was looking for a way to create another dataset for the multi-eval dataset tests.

Reply:
oh cool! thanks for explaining and fixing!